_posts/2018-07-05-client-provisioning-methods.adoc

```diff
@@ -6,45 +6,40 @@
 :icons: font
 :toc: macro
 
-// TODO MERLIN: Explain the difference between different provisioning methods, why you might want to do one or another, and what steps you need to take for each.
-
-
-
 In OTA Connect, the provisioning process ensures that the device has a certificate and a unique identifier. After a device has been provisioned, it can receive software updates. OTA Connect supports two types of provisioning:
 
-* *Automatic provisioning*
+* *Provisioning with shared credentials*
 
 Devices use temporary provisioning credentials to request permanent credentials from the server. You download the temporary provisioning credentials from the OTA Connect web app and install them on a device. Once the device registers with the server for the first time, the server assigns permanent credentials to the device. The device then uses these permanent credentials for all future transactions.
```

Comment:
Indentation is not right compared to the next paragrah

Reply:
Fixed